services/static-webserver/client/source/class/osparc/navigation/Manuals.js

```diff
@@ -53,18 +53,101 @@ qx.Class.define("osparc.navigation.Manuals", {
       return manuals;
     },
 
+    __openGithubIssueInfoDialog: function() {
+      const issueConfirmationWindow = new osparc.ui.window.Dialog("Information", null,
+        qx.locale.Manager.tr("To create an issue in GitHub, you must have an account in GitHub and be already logged-in.")
```

Comment:
Q: have you ever tried to activate the translation?

Reply:
4 years ago... then we dropped it, but the frontend fully supports it.

Reply:
i think it is a shame. Perhaps we should bring that up again in the s4l meeting?